tests/chainerx_tests/unit_tests/routines_tests/test_linalg.py

```diff
@@ -9,6 +9,8 @@
 from chainerx_tests import dtype_utils
 from chainerx_tests import op_utils
 
+from distutils.version import LooseVersion
```

Comment:
A minor suggestion (although I just noticed we're already using LooseVersion elsewhere) to use NumpyVersion instead to check the NumPy version as in https://github.com/chainer/chainer/blob/master/chainer/functions/array/get_item.py#L13.

Reply:
I've changed that. Thank you!